DALK.PL_ANALYZER/Scripts/modernizr-site.js

```diff
@@ -12,8 +12,40 @@
     }
 }
 
+function verticalAlignAllElements()
+{
+    setHeightForMvpAndDesc();
+    setMarginInNotPlayedMatchNavigation();
+}
+
 function setHeightForMvpAndDesc() {
     setMaxHeightForAllElementsInClass(".match__mvp");
     setMaxHeightForAllElementsInClass(".match__description");
 }
 
+function setMarginInNotPlayedMatchNavigation() {
+    var allNotPlayedMatch = document.getElementsByClassName("match__notPlayed");
```

Comment:
Its a good practice to separate html classes that are used to styliing and/or are used for developing. So plase add prefix js-blablabla or css-blablabla (just one should be enaugh) to point that this class is used for develop/style.
This way dev-purpose classes will not be used for styling and vice-versa

Reply:
DONE. Added -js prefix.